### PR TITLE
Correct typos in developer documentation

### PR DESCRIPTION
**Pull request by jugglinmike**
### Link to issue number:
n/a
### Summary of the issue:
The developer documentation has a few spelling errors
### Description of how this pull request fixes the issue:
This patch corrects the spelling errors
### Testing strategy:
n/a
### Known issues with pull request:
none
### Change log entries:
New features
Changes
Bug fixes
For Developers

### Code Review Checklist:

- [ ] Pull Request description:
  - description is up to date
  - change log entries
- [ ] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [ ] API is compatible with existing add-ons.
- [ ] Documentation:
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [ ] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English

### CODE REVIEW

**Comment by lukaszgo1** [comment]
Hi @jugglinmike  This looks great! Any particular reason why this is still a draft? Note tat if you want this to be reviewed by NV Access developers you should mark it as ready for review.

**Comment by jugglinmike** [comment]
Thanks for the heads up, @lukaszgo1! The draft status must have been a mistake on my part.

**Comment by CyrilleB79** [comment]
Hello

I can see that you have modified `user_docs/pt_BR/developerGuide.t2t`.

First of all, I did not know that translated versions of the developer's guided existed in NVDA. Looking in the `user_docs` folder, I can see that there is a version for German (de) and Portuguese Bresil (pt_BR). Last commit on these files are 10 year old.
So I think they are not maintained anymore.

I can see two options:
1. Do not modify these files in these PRs; I guess it is the job of translators to keep track on the modifications in the English Dev Guide and report them translated to the locale dev guide.
2. Remove these files from NVDA since they are not maintained anymore; maybe there is also something in the scons scripts to generate the .html from the .t2t file and that should also be removed.

In any case, I think that only the English file should be modified.
IMO, option 2 should be chosen, either in this PR or in a separate PR.
Let's hear NVAccess thoughts on this topic.

Cc @bdorer, @mhameed who have committed on these files.
Also wanted to cc David Parduhn, Rene Linke and Peter Vagne, but I do not have their GitHub user name (if ever they have one).

File: user_docs/pt_BR/developerGuide.t2t [hunk 1]
```diff
@@ -325,7 +325,7 @@ Plugins make use of NVDA Objects in three particular ways:
 - Most events that plugins receive take an argument which is the NVDA Object on which the event occurred.
 For example, event_gainFocus takes the NVDA Object that represents the control gaining focus.
 - Scripts, events or other code may fetch objects of interest such as the NVDA Object with focus, NVDA's current navigator object, or perhaps the Desktop NVDA Object.
-The code may then retreave information from that object or perhaps even retreave another object related to it (e.g. its parent, first child, etc.).
+The code may then retrieve information from that object or perhaps even retrieve another object related to it (e.g. its parent, first child, etc.).
```
Comment (seanbudd) [inline user_docs/pt_BR/developerGuide.t2t:328]: Please remove the changes from this file

Comment (jugglinmike) [reply user_docs/pt_BR/developerGuide.t2t]: You got it!

**Comment by seanbudd** [comment]
@CyrilleB79, I agree that option 2 should be considered in a separate PR. I would want to confirm that these should be removed with other developers, and investigate why they were added and if anybody is relying on them.
Thanks @jugglinmike for reverting those changes in this PR.

**Review by seanbudd — APPROVED** [review]

**Comment by CyrilleB79** [comment]
> @CyrilleB79, I agree that option 2 should be considered in a separate PR. I would want to confirm that these should be removed with other developers, and investigate why they were added and if anybody is relying on them. Thanks 

FYI, I have opened #13333 to track this.

**Pull request merged**